contentcuration/contentcuration/frontend/shared/views/__tests__/masteryDropdown.spec.js

```diff
@@ -89,14 +88,6 @@ describe('masteryDropdown', () => {
       });
     });
   });
-  describe('mastery model info modal', () => {
-    it('should open the info modal when button is clicked', () => {
```

Comment:
I removed this test because this behavior is the responsibility of `InfoModal` and is already tested there.